decomposed/extensions/MDDE_LDM/Profile/Model/Extended_Attributes/mdde_feature_business_rule_aggregate.xml

```diff
@@ -0,0 +1,9 @@
+<o:ExtendedAttributeTargetItem Id="52F0CC72-6B0C-44B8-A875-5DC31F6A52BA">
+<a:ObjectID>52F0CC72-6B0C-44B8-A875-5DC31F6A52BA</a:ObjectID>
+<a:Name>mdde_feature_business_rule_aggregate</a:Name>

```

Comment:
Could you follow the convention of having every word with a captial (except in prefix)? And make business_rule one word: BusinessRule (like in the stereotype)?

mdde_feature_business_rule_aggregate > mdde_Feature_BusinessRule_Aggregate

This should happen for all 5 new extended attributes. Not the placed where the extended attribute is used is not automatically updated with the new name. So you can consider doing the edit in Xml format. What you prefer.